src/System.Console/tests/Configurations.props

```diff
@@ -1,8 +1,10 @@
 ﻿<Project>
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard;
-      netstandard-Windows_NT;
+      netcoreapp;
+      netcoreapp-Windows_NT;
+      uap;
```

Comment:
This os agnostic uap configuration doesn't make sense. Uap can only build for Windows_NT.

Reply:
Same comment as below.